data/stoa0171/stoa001/stoa0171.stoa001.opp-lat3.xml

```diff
@@ -46,28 +48,28 @@
               <editor>
                 <persName>
                   <name xml:lang="lat">Samuel Brandt</name>
```

Comment:
Not sure why we would have Samuel Brandt twice and tagged as two languages? I would keep one, and remove the language attribute. Except where OCT or others Latinize modern names (and we don't use those forms), I don't think it's standard practice to tag a name with a language.

Reply:
You know I've been wondering about this. This was standard practice I think just for these files back in the day.  I think as I move forward I will just keep one version of the name and remove the language attribute since that corresponds to current practice.

Reply:
It's a legitimate question, but my feeling is that we want the headers light and that this sort of thing is linked elsewhere. I guess it could be an issue if someone were to search on the Latinized editors' names, but we never adhered to it before. As with the Perseus files, it brings up the question of linking to WorldCat in the header itself instead of through other tools.